### PR TITLE
NX: devextreme package scripts configs

### PR DESCRIPTION
**Pull request by iBat**

### CODE REVIEW

**Review by alexslavr — DISMISSED** [review]

File: packages/devextreme/project.json [hunk 1]
```diff
@@ -11,23 +11,151 @@
...
+      ],
+      "cache": true
+    },
+    "compile:r": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "compile:r"
+      },
+      "inputs": [
+        "{projectRoot}/js/renovation/**/*"
```
Comment (pomahtri) [inline packages/devextreme/project.json]: include please tsconfig

File: packages/devextreme/project.json [hunk 2]
```diff
@@ -11,23 +11,151 @@
...
+      ],
+      "cache": true
+    },
+    "lint-js": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "lint-js"
+      },
+      "inputs": [
+        "{projectRoot}/**/*.js"
```
Comment (pomahtri) [inline packages/devextreme/project.json]: include please eslint config (nested too)

File: packages/devextreme/project.json [hunk 3]
```diff
@@ -11,23 +11,151 @@
...
+    },
+    "lint-dts": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "lint-dts"
+      },
+      "inputs": [
+        { "env": "DEBUG" },
+        "{projectRoot}/**/*.d.ts",
+        "!{projectRoot}/js/renovation/**/*"
```
Comment (pomahtri) [inline packages/devextreme/project.json:85]: eslint config + tsconfig

File: packages/devextreme/project.json [hunk 4]
```diff
@@ -11,23 +11,151 @@
...
+      "executor": "nx:run-script",
+      "options": {
+        "script": "lint-ts"
+      },
+      "inputs": [
+        { "env": "DEBUG" },
+        "{projectRoot}/**/*.ts?(x)",
+        "!{projectRoot}/**/*.d.ts",
+        "!{projectRoot}/js/renovation/**/*",
+        "!{projectRoot}/testing/renovation/**/*"
```
Comment (pomahtri) [inline packages/devextreme/project.json:111]: eslint

File: packages/devextreme/project.json [hunk 5]
```diff
@@ -11,23 +11,151 @@
...
+      ],
+      "cache": true
+    },
+    "lint-renovation": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "lint-renovation"
+      },
+      "inputs": [
+        "{projectRoot}/js/renovation/**/*.ts?(x)"
```
Comment (pomahtri) [inline packages/devextreme/project.json]: eslint

File: packages/devextreme/project.json [hunk 6]
```diff
@@ -11,23 +11,151 @@
...
+      ],
+      "cache": true
+    },
+    "test-jest": {
+      "executor": "nx:run-script",
+      "options": {
+        "script": "test-jest"
+      },
+      "inputs": [
+        "{projectRoot}/js/**/*"
```
Comment (pomahtri) [inline packages/devextreme/project.json]: jest config

**Review by pomahtri — APPROVED** [review]

**Pull request merged**